.github/workflows/test_and_release.yml

```diff
@@ -0,0 +1,127 @@
+name: Test and Release
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+  pull_request:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+-dev
+
+jobs:
+  test:
+    name: Run SDK tests
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Enable NPM cache
+        uses: actions/cache@v2
+        with:
+          path: '~/.npm'
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install NPM dependencies
+        run: npm ci
+
+      - name: Check NPM package lock version is updated
+        run: npm run check-package # TODO: does this exist as an action already?
+
+      - name: Cache Docker layers
+        uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+        with:
+          concurrency: 8
+
+      - name: Load workflow dotenv file
+        uses: c-py/action-dotenv-to-setenv@v3
+        with:
+          env-file: .github/workflows/.env
+
+      - name: Set compatible branches for Platform components
+        uses: dashevo/gh-action-platform-branches@master
+        id: platform-branches
+        with:
+          override-major-version: 0
+          override-dashmate-branch: ${{ env.DASHMATE_BRANCH }}
+          override-testsuite-branch: ${{ env.TEST_SUITE_BRANCH }}
+
+      - name: Start Dash local network
+        uses: dashevo/gh-action-start-local-network@master
+        with:
+          actions-cache-url: ${{ steps.platform-branches.outputs.cache-url }}
+          actions-cache-token: ${{ steps.platform-branches.outputs.cache-token }}
+          drive-branch: ${{ env.DRIVE_BRANCH }}
+          dapi-branch: ${{ env.DAPI_BRANCH }}
+          dashmate-branch: ${{ steps.platform-branches.outputs.dashmate-branch }}
+          sdk-branch: ${{ steps.platform-branches.outputs.current-branch }}
+
+      - name: Create dotenv file
+        run: |
+          echo "DAPI_SEED=127.0.0.1" > .env
+          echo "FAUCET_PRIVATE_KEY=${{ env.faucet-private-key }}" >> .env
+          echo "NETWORK=regtest" >> .env
+          echo "DPNS_CONTRACT_ID=${{ env.dpns-contract-id }}" >> .env
+
+      - name: Run SDK tests
+        run: npm run test
+
+      - name: Run Platform Test Suite
+        uses: dashevo/gh-action-run-platform-test-suite@master
+        with:
+          sdk-install: ${{ steps.platform-branches.outputs.current-branch }}
```

Comment:
sdk-branch?

Reply:
I think `sdk-install` might be more accurate here, because it is modifying and passing it to the `npm-install` variable of platform-test-suite [here](https://github.com/dashevo/gh-action-run-platform-test-suite/blob/master/action.yml#L43). In theory we could be installing any NPM package here, just in this case it happens to be the SDK.

Reply:
Yeah, but since we changed the action https://github.com/dashevo/gh-action-run-platform-test-suite/blob/master/action.yml#L43 it accepts only branch for SDK. And it would be less confusing to have the same names for inputs in start local network and run test suite